### PR TITLE
add a function that separates tropics extratropics

### PR DESCRIPTION
**Pull request by Datseris**

### CODE REVIEW

File: src/physical_dimensions/spatial.jl [hunk 1]
```diff
@@ -232,3 +232,24 @@ end
...
+"""
+    tropics_extratropics(A::ClimArray) → tropics, extratropics
+Separate the given array into two arrays: one having latitudes ℓ ∈ [-30, 30], and one
+having all other remaining latitudes.
+"""
+tropics_extratropics(A, args...) = 
+tropics_extratropics(spacestructure(A), A, args...)
+
+function tropics_extratropics(::LonLatGrid, A)
+    tropics = A[Lat(Between(-30, 30))]
```
Comment (Balinus) [inline src/physical_dimensions/spatial.jl:248]: Might be useful to add keywords for the bounding box?  I mean, perhaps a user would want to separate between -25, +25 for a reason?

Comment (Datseris) [reply src/physical_dimensions/spatial.jl:248]: Yeap sounds good. Do you want to do a PR that adds this? would be nice to get more commits with your name!

Comment (Balinus) [reply src/physical_dimensions/spatial.jl:248]: Sure :) 

What would be the best approach here? A simple keyword like `tropics = Lat(Between(-30, 30))` ? And derive the extratropics based on this?

Comment (Datseris) [reply src/physical_dimensions/spatial.jl:248]: I think an even simpler keyword `lat = 30`, and the `Between` gets `-lat, lat`.

Comment (Balinus) [reply src/physical_dimensions/spatial.jl:248]: ok! Will do that.

Comment (Balinus) [reply src/physical_dimensions/spatial.jl:248]: Done in #72

**Pull request merged**